packages/pytest-simcore/src/pytest_simcore/docker_swarm.py

```diff
@@ -77,9 +78,13 @@ def _wait_for_services(docker_client: docker.client.DockerClient) -> None:
         if service.tasks():
             sorted_tasks = sorted(service.tasks(), key=by_task_update)
             task = sorted_tasks[-1]
-            if task["Status"]["State"].upper() not in pre_states:
-                if not task["Status"]["State"].upper() == "RUNNING":
-                    raise Exception(f"service {service.name} not running")
+            task_state = task["Status"]["State"].upper()
+            if task_state not in pre_states:

```

Comment:
```suggestion
            if (task_state := task["Status"]["State"].upper()) not in pre_states:

```

Reply:
Unfortunately we cannot use 3.8 syntax in pytest-simcore because it is also used in the director service, which is frozen 3.6